src/fft.js

```diff
@@ -467,9 +467,7 @@ define(function (require) {
    *                               Defaults to 0.8.
    */
   p5.FFT.prototype.smooth = function(s) {
-    if (s) {
-      this.smoothing = s;
-    }
+    this.smoothing = s;
     this.analyser.smoothingTimeConstant = s;
```

Comment:
I'm not sure if I fully understand what you're trying to explain... 🤔

Shouldn't `this.smoothing` be always equal to `this.analyser.smoothingTimeConstant`?

I've updated my comment to correctly align with the meaning that I had intended, but not correctly put across. :joy: